rero_ils/modules/locations/jsonschemas/locations/location-v0.0.1.json

```diff
@@ -112,6 +116,63 @@
           "pattern": "^https://ils.rero.ch/api/libraries/.*?$"
         }
       }
+    },
+    "contact_email": {
+      "title": "Contact email",
```

Comment:
description is missing. What is the purpose of this field?

Reply:
I added description. This field could be used to send email notification to a generic email address (by example, when a item of this location is requested -- it's a use case at UCLouvain)